Kernel/Clause.cpp

```diff
@@ -75,7 +77,8 @@ Clause::Clause(unsigned length,const Inference& inf)
     _reductionTimestamp(0),
     _literalPositions(0),
     _numActiveSplits(0),
-    _auxTimestamp(0)
+    _auxTimestamp(0),
+    _supData(0)
 {
   // MS: TODO: not sure if this belongs here and whether EXTENSIONALITY_AXIOM input types ever appear anywhere (as a vampire-extension TPTP formula role)
```

Comment:
Since this is here: I think `EXTENSIONALITY_AXIOM` might be completely useless and I've never seen it in TPTP input: was it perhaps used for experiments wrt "Extensional Crisis and Proving Identity"? If so it can probably go.

Reply:
The input type EXTENSIONALITY_AXIOM could go. It was out custom tptp extension, IIRC.

The actual trick (which may be fed ext. axioms also through "fishing") is still doing something an appears in strategies...